tools/base/envoy_python.bzl

```diff
@@ -67,4 +69,33 @@ def envoy_py_binary(
         visibility = visibility,
     )
 
-    envoy_py_test(name, package, visibility, envoy_prefix = envoy_prefix)
+    if test:
+        envoy_py_test(name, package, visibility, envoy_prefix = envoy_prefix)
+
+def envoy_py_script(
```

Comment:
Please add some docs explaining what this is offering compared to `envoy_py_binary`. Otherwise LGTM.